--- a/lib/dfa.h
+++ b/lib/dfa.h
@@ -89,7 +89,7 @@ enum
        says \] and \} have undefined interpretation, platforms
        reliably ignore those stray backlashes and warning about them
        would likely cause more trouble than it's worth.  */
-    DFA_STRAY_BACKSLASH_WARN = 1 << 3,
+    DFA_STRAY_BACKSLASH_WARN = 0,
 
     /* Warn about * appearing out of context at the start of an
        expression or subexpression.  */
