--- ./minissdp.c-orig	2021-07-24 19:41:14.735550487 +0000
+++ ./minissdp.c	2021-07-24 19:42:49.443074366 +0000
@@ -650,7 +650,7 @@ ProcessSSDPRequest(struct event *ev)
 			DPRINTF(E_INFO, L_SSDP, "WARNING: Ignoring invalid SSDP M-SEARCH from %s [bad source port %d]\n",
 				inet_ntoa(sendername.sin_addr), ntohs(sendername.sin_port));
 		}
-		else if (!man || (strncmp(man, "\"ssdp:discover\"", 15) != 0))
+		else if (0)
 		{
 			DPRINTF(E_INFO, L_SSDP, "WARNING: Ignoring invalid SSDP M-SEARCH from %s [bad %s header '%.*s']\n",
 				inet_ntoa(sendername.sin_addr), "MAN", man_len, man);
