--- ./Makefile.in-orig	2022-07-28 17:34:23.000000000 +0200
+++ ./Makefile.in	2022-09-08 00:53:35.067294761 +0200
@@ -914,7 +914,7 @@ EXTRA_DIST = README.md COPYING.LGPL FAQ.
 	$(am__append_6) $(am__append_8) $(am__append_14) \
 	lib/libcryptsetup.pc.in lib/libcryptsetup.sym \
 	tokens/libcryptsetup-token.sym
-SUBDIRS = po tests
+SUBDIRS =
 CLEANFILES = 
 DISTCLEAN_TARGETS = scripts/cryptsetup.conf
 AM_CPPFLAGS = \
--- ./Makefile.am-orig	2022-07-28 12:53:44.000000000 +0200
+++ ./Makefile.am	2022-09-08 00:53:47.029491837 +0200
@@ -1,5 +1,5 @@
 EXTRA_DIST = README.md COPYING.LGPL FAQ.md docs misc autogen.sh
-SUBDIRS = po tests
+SUBDIRS =
 CLEANFILES =
 DISTCLEAN_TARGETS =
 
