--- ./configure-orig	2019-06-10 16:38:01.000000000 +0200
+++ ./configure	2019-12-25 17:07:05.877892857 +0100
@@ -19992,10 +19992,7 @@ cat >>confdefs.h <<_ACEOF
 _ACEOF
 
       if test -f $dir/include/gssapi/gssapi_krb5.h -a \
-                \( -f $dir/lib/libgssapi_krb5.a -o \
-                   -f $dir/lib64/libgssapi_krb5.a -o \
-                   -f $dir/lib64/libgssapi_krb5.so -o \
-                   -f $dir/lib/libgssapi_krb5.so \) ; then
+                \( -f $dir/lib/libgssapi_krb5.so \) ; then
 
 $as_echo "#define HAVE_KRB5 1" >>confdefs.h
 
--- ./utils/nfsdcld/legacy.c-orig	2019-06-10 16:34:37.000000000 +0200
+++ ./utils/nfsdcld/legacy.c	2019-12-26 14:00:09.949758475 +0100
@@ -31,6 +31,10 @@
 
 #define NFSD_RECDIR_FILE "/proc/fs/nfsd/nfsv4recoverydir"
 
+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
+
 /*
  * Loads client records from the v4recovery directory into the database.
  * Records are prefixed with the string "hash:" and include the '\0' byte.
--- ./utils/idmapd/idmapd.c-orig	2019-06-10 16:34:37.000000000 +0200
+++ ./utils/idmapd/idmapd.c	2019-12-26 14:43:39.697981705 +0100
@@ -88,7 +88,7 @@
 #endif
 
 #ifndef NFS4NOBODY_GROUP
-#define NFS4NOBODY_GROUP "nobody"
+#define NFS4NOBODY_GROUP "nogroup"
 #endif
 
 /* From Niels */
